Exception.log_exceptions = false
